check/index.php

```diff
@@ -221,30 +221,35 @@
 
 // if prerequisite or next checks are inserted into db successfully, 
 // $javascript_run_now is set to refresh parent window 
-if (isset($javascript_run_now)) $savant->assign('javascript_run_now', $javascript_run_now);
-
-$savant->assign('check_rows', $check_rows);
-$savant->assign('all_html_tags', $checksDAO->getAllHtmlTags());
-$savant->assign('results_per_page', $results_per_page);
-$savant->assign('num_results', $num_results);
-$savant->assign('col_counts', $col_counts);
-$savant->assign('page',$page);
-$savant->assign('page_string', $page_string);
-$savant->assign('orders', $orders);
-$savant->assign('order', $order);
-$savant->assign('col', $col);
+if (isset($javascript_run_now)) $plate['javascript_run_now'] = $javascript_run_now;
+

```

Comment:
Remove stray empty lines. Same at line 240 and 246, 267. This comment applies across the pull request.